srv/pom.xml

```diff
@@ -25,6 +25,12 @@
       <artifactId>cds-starter-cloudfoundry</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>com.sap.cds</groupId>
+      <!-- Use only for development. Remove in production! -->
```

Comment:
At least to my understanding the dev dashboard is not active in production (with Spring Boot profile "cloud"). We could however, also introduce a Maven profile where this dependency is active and by default it is not active.